torch/_dynamo/variables/builder.py

```diff
@@ -420,6 +420,20 @@ def index_source(key):
                 source=self.source,
                 guards=make_guards(GuardBuilder.FUNCTION_MATCH),
             )
+        elif (
+            not is_allowed(value)
```

Comment:
Can we just make `is_allowed()` return True for these?

Should be able to modify:
https://github.com/pytorch/pytorch/blob/74592a43d0d33a6c809fdcfc20249e1c93e7216e/torch/_dynamo/allowed_functions.py#L129

with something like:
```
for name in dir(torch.Tensor):
   method = getattr(torch.Tensor, name)
   if isinstance(method, types.MethodDescriptorType):
       torch_object_ids[id(method)] = f"torch.Tensor.{name}"
```

Reply:
Hmm, `torch.Tensor.{name}` are method descriptors not module or method, hence doesn't have `__module__` attribute. This issue will propagate to when generating fx graph and finding the module of the method: 

https://github.com/pytorch/pytorch/blob/e7ace1ff930db30deb907c753dc9e41b26f2586f/torch/fx/node.py#L40

Reply:
Could we change FX to handle that?  I suspect it just involves adding a case to the code printer.

Reply:
Thanks @jansel, your suspect was right. Please help have another look.